### PR TITLE
Cleanup unnecessary rhel cases

### PR DESCRIPTION
**Pull request by marusak**
Both rhel-atomic and continuous atomic are not a thing anymore.

### CODE REVIEW

File: test/check-ostree [hunk 1]
```diff
@@ -41,25 +41,15 @@ CHECKOUT_LOCATION = "/var/local-tree"
...
-    if m.image == "rhel-atomic":
-        return "rhel-atomic-host"
-    elif m.image == "fedora-coreos":
+    if m.image == "fedora-coreos":
         return "fedora-coreos"
-    elif 'continuous' in m.image:
-        return 'centos-atomic-host'
-    else:
-        return "fedora-atomic"
+    return "fedora-atomic"
```
Comment (martinpitt) [inline test/check-ostree:52]: Fedora Atomic also hasn't been a thing for a while. Want to clean that up as well while you are at it? I think this should rather `raise NotImplementedError` here.

**Review by martinpitt — COMMENTED** [review]
Cheers!

File: test/check-ostree (hunk 1 above)
Comment (marusak) [reply test/check-ostree:52]: I was wondering what should be the fallback. Is there any chance that we will introduce anything else? If not, then I can just replace calls to this function with directly adding `fedora-coreos` into the code.

Comment (martinpitt) [reply test/check-ostree:52]: Right now there's only fedora-coreos. RHEL coreos isn't a standalone supported product. We may add this at some point, but as there is nothing planned right now, I'd just make it as simple as possible, and we can reintroduce an indirection if/when we need it.

File: test/check-ostree [hunk 2]
```diff
@@ -247,7 +230,7 @@ class OstreeRestartCase(MachineCase):
 
         # Check update data
         b.wait_visible("{0} div.tree".format(get_list_item(1)))
-        b.wait_in_text("{0} .os".format(get_list_item(1)), name)
+        b.wait_in_text("{0} .os".format(get_list_item(1)), "fedora-coreos")
```
Comment (martinpitt) [inline test/check-ostree:233]: ah, I didn't think of that usage here -- that repeats the OS name a lot here. But *shrug*, we can re-factorize it later on as well.

**Review by martinpitt — APPROVED** [review]
Thank you!

**Pull request merged**